shell-control/run-agent-cli.sh

```
@@ -0,0 +1,61 @@
+
+if [ ! -f "${ENV_FILE}" ]; then
+    echo "Error: ${ENV_FILE} not found. Please create it from .env.example" >&2
+    exit 1
+fi
+
+build() {
+    docker build -q --target python-base --tag ${IMAGE} --file agent/Dockerfile ./agent
+}
+
+prepare_workdir() {
+    readonly WORK_DIR="work/run-$(date +%s)"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)